src/posix/unix/i_specialpaths.cpp

```diff
@@ -130,7 +154,8 @@ FString M_GetCachePath(bool create)
 {
 	// Don't use GAME_DIR and such so that ZDoom and its child ports can
 	// share the node cache.
-	FString path = NicePath("$HOME/.config/zdoom/cache");
+	FString path = GetEnvironmentVariable("XDG_CONFIG_HOME", FString("$HOME/.config/"));
+	path += "zdoom/cache/";
```

Comment:
This needs a leading slash, in cae the env variable doesn't contain a trailing slash